test/azure/legacy/Expected/AcceptanceTests/CustomUrlPaging/generated_samples/paging_get_pages_partial_url.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from custombaseurlpaging import AutoRestParameterizedHostTestPagingClient
+
+"""
+The sample just shows how to use the method and may not run successfully.
+# PREREQUISITES
+    pip install AutoRestParameterizedHostTestPagingClient
+# USAGE
+    python paging_get_pages_partial_url.py
```

Comment:
if this is an AAD client, then we should check if users have set the AZURE_TENANT_ID etc env vars.

We can also check for AZURE_KEY_CREDENTIAL if the client has `AzureKeyCredential` for its credential

Reply:
good point. I want users to see the how to use client directly instead of too many details so I put the check at [bottom](). What do you think about that? @iscai-msft